internal/corazawaf/transaction.go

```diff
@@ -865,10 +865,10 @@ func (tx *Transaction) AuditLog() *loggers.AuditLog {
 	return al
 }
 
-// Clean the transaction after phase 5
+// clean the transaction after phase 5
 // This method helps the GC to clean up the transaction faster and release resources
 // It also allows caches the transaction back into the sync.Pool
-func (tx *Transaction) Clean() error {
+func (tx *Transaction) clean() error {
```

Comment:
Thanks - I had mostly kept the method to reduce the size of the refactoring PR, we can just remove this method and inline into `Close` I think